newrelic/resource_newrelic_alert_channel.go

```diff
@@ -152,10 +152,26 @@ func resourceNewRelicAlertChannel() *schema.Resource {
 							Description: "The key for integrating with VictorOps.",
 						},
 						"include_json_attachment": {
-							Type:        schema.TypeString,
-							Optional:    true,
-							ForceNew:    true,
-							Description: "0 or 1. Flag for whether or not to attach a JSON document containing information about the associated alert to the email that is sent to recipients.",
+							Type:         schema.TypeString,
+							Optional:     true,
+							ForceNew:     true,
+							Description:  "true or false. Flag for whether or not to attach a JSON document containing information about the associated alert to the email that is sent to recipients.",
+							ValidateFunc: validation.StringInSlice([]string{"true", "false", "1", "0"}, true),
+							DiffSuppressFunc: func(k, old, current string, d *schema.ResourceData) bool {
```

Comment:
Ignore current `1`s  or `0`s

Reply:
Not a blocker, but just putting the thought out there... wondering if theres any difference between using `DiffSuppressFunc` vs `StateFunc` where `StateFunc` converts the boolean value to `0` or `1` to store it in the state. It probably doesn't make a difference in this case 😎